README.md

```diff
@@ -109,13 +109,27 @@ You probably need to set some environment variables for the client to be useful.
 
 ## Troubleshooting
 
+### I'm getting compilation errors
+
+Make sure you are using NodeJS 12. This may involve installing an LTS version from a package manager rather than the latest stable version.
+
+### The server can't create an HID device
+
+Run the server as root.
+
 ### The UI shows more sensors than I have
 
 The firmware reads all 12 ADC pins on the Teensy. You may get fake readings from pins that have no sensors plugged in and are not grounded. If pressing a sensor causes more than one value to go up:
   - Determine which pins are unused. You can use this [pinout](https://www.pjrc.com/teensy/card2a.pdf). The sensors follow the pin order, e.g. ADC0 = 1, ADC1 = 2...
   - Press settings and disable the corresponding sensors in the UI, or
   - Ground the unused inputs on the board.
 
+### Stepmania isn't registering my inputs
+
+Make sure you have the proper drivers specified in Preferences.ini, e.g.
+
+`InputDrivers=X11,HID`
```

Comment:
Hmm. From what I remember you shouldn't use X11 driver for pad input. But it's a good point that you should have proper `InputDriver` settings anyway. I'll check what we use in our arcade at a better time.